### ISSUE TITLE
Standardize data format

### ISSUE THREAD

**Issue by philippedm** [dd7ler/iris-processing#9]
Many functions save the spot data in different formats.  It would be much easier if it was standardized.  something like data(rows, columns, timesteps) would be great and intuitive.

**Comment by Giuliariga** [comment]
I agree. Actually I'm using row_timesteps_coumns, but probably what you are suggesting make more sense.

**Comment by Giuliariga** [comment]
- row, timesteps, column
